### PR TITLE
Update Mission Statement

### PR DESCRIPTION
**Pull request by sierrafontaine**
Open to suggestion here, maybe "data professionals" - but got some feedback that "data analyst" doesn't capture all of the people that use dbt, so it may be time for a refresh!

### CODE REVIEW

**Review by jthandy — CHANGES_REQUESTED** [review]
I don't think "data humans" is very well-defined. What we must avoid with our mission and values is for them to be "squishy"--they have to be extremely clear. In order to say "yes this is the mission" or "no this is not the mission" that clarity is critical. 

It then becomes critical to know whom we are empowering. Who is a "data human"? This is unanswerable. We could define what we mean by this, but going into a lengthy definition defeats one of the purposes of the mission: to succinctly, memorably communicate about purpose.

So IMO "data human" is a no go.

I'm open to "data professionals". I think it is specific, clear, and memorable. I also think that it is _true_. I agree with the core assertion of this PR that "analysts" is not a sufficiently-wide grouping to target our mission at any more. This might have been the correct answer ~3 years ago when this version was merged, but we've evolved a lot since then.

**Comment by lostmygithubaccount** [comment]
+1 for data professionals. in ML product land we used "ML professionals" for the same reasoning -- there are a lot of titles (data scientist, ML engineer, ML/AI researcher, data wranglers, ML app devs, MLOps engineer, ...) and it's much easier to lump them all together into a neutral term when talking generally about users

**Comment by jthandy** [comment]
@sierrafontaine please change this to "data practitioner" and then we can merge. thanks!

File: mission.md [hunk 1]
```diff
@@ -1,3 +1,3 @@
 # dbt Labs Mission Statement
 
-To empower analysts to create and disseminate organizational knowledge.
+To empower data professionals to create and disseminate organizational knowledge.
```
[the remark this answers is not in the document]
Comment (brian-gillet) [reply mission.md:3]: Misspelled practitioners @dataders - can you please update the spelling in the commit?

[the remark this answers is not in the document]
Comment (dataders) [reply mission.md:3]: whoops! I thought it looked funny!
```suggestion
To empower data practitioners to create and disseminate organizational knowledge.
```

**Review by ericalouie — APPROVED** [review]
LGTM 🥳

**Pull request merged**